frontend/public/components/RBAC/role.jsx

```diff
@@ -249,12 +278,37 @@ export const RolesDetailsPage = (props) => {
         { href: 'bindings', name: t('role~RoleBindings'), component: BindingsForRolePage },
       ]}
       menuActions={menuActions}
+      breadcrumbsFor={
+        props.kind === 'ClusterRole'
+          ? () => getBreadcrumbs('Role', props.kindObj, props.match)
+          : null
```

Comment:
I'm confused about this check. Shouldn't we go to the same page for both cluster roles and roles?

Reply:
Yes, and that same page will be the Roles list, as opposed to the previous ClusterRoles list view.

Reply:
OK, I see how it works. But would it be simpler to just make this...?

```ts
breadcrumbsFor({() => getBreadcrumbs('Role', props.kindObj, props.match)}
```

If I'm not mistaken, the behavior is the same, but with less logic. You're also sure in that case that both cases always go to the same place. (If the default breadcrumb changes for some reason, that would no longer be true.)

Reply:
agree, logic is the same, updated -thanks